sos/plugins/origin.py

```diff
@@ -155,15 +173,14 @@ def setup(self):
         if self.is_node():
             self.add_copy_spec([
                 self.node_cfg,
+                self.node_kubeconfig,
+                self.node_certs,
+                self.node_client_ca,
+                self.bootstrap_node_cfg,
+                self.bootstrap_kubeconfig,
                 os.path.join(self.node_base_dir, "*.crt"),
             ])
 
-            node_kubecfg = self.get_node_kubecfg()
```

Comment:
If you're deleting the only call to this method please also delete the method definition.